ta/Makefile

```diff
@@ -5,6 +5,8 @@ ifeq ($O,)
 $(error output path should be specified when calling this makefile)
 endif
 
+LDFLAGS=
```

Comment:
In fact, it's not the LDFLAGS that makes the build fail, it is the other changes in `xtest/Makefile` that is the culprit. The reason is that there are a few of the variables which already are implicitly set from the start:
```
CC: cc
CPP: cc -E
LD: ld
AR: ar
```
So, those will always be set when this file is entered and therefore the right side after `?=` will __never__ be used.

https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html